src/js/components/Header.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Header extends React.Component {
+
+  getIcon(icon, className) {
+    if (icon == null) {
+      return null;
+    }
+
+    return (
+      <div className="media-object-item">
+        <div className={className}>
+          {icon}
+        </div>
+      </div>
+    );
+  }
+
+  getTitle(title, className) {
+    if (title == null) {
+      return null;
+    }
+
+    return (
+      <h1 className={className}>
+        {title}
+      </h1>
+    );
+  }
+
+  getSubTitle(subTitle) {
+    if (subTitle == null) {
+      return null;
+    }
+
+    return (<div>{subTitle}</div>);
+  }
+
+  render() {
+    let props = this.props;
+
+    return (
+      <div className={props.className}>
+        <div className={props.dividerClassName}>
+          <div className={props.mediaWrapperClassName}>
+            <div className="media-object flex-box flex-box-align-vertical-center">
+              {this.getIcon(props.icon, props.iconClassName)}
```

Comment:
What do you think about making these `this.get..` methods variables by `let props`? Personal preference but it might make this render method a little easier on the eye :)